### ISSUE TITLE
This extension cannot be found in jupterlab, error reported after installation through cmd, internal error 500 ,why?

### ISSUE THREAD

**Issue by x83443647629** [pbugnion/jupyterlab-sql#127]
This extension cannot be found in jupterlab, error reported after installation through cmd, internal error 500 ,why?

**Comment by kamilm** [comment]
I agree, it took me a while to find out that this was the culprit. It only started after a recent conda upgrade. 

JupyterLab v1.2.6

**Comment by jseabold** [comment]
Duplicated by #130 and #131.
